price-feeder/oracle/oracle.go

```diff
@@ -293,6 +293,12 @@ func (o *Oracle) checkAcceptList(params oracletypes.Params) {
 			o.logger.Warn().Str("denom", symbol).Msg("price missing for required denom")
 		}
 	}
+	for symbol := range o.prices {
```

Comment:
Not sure if we want this logic to be in `o.SetPrices()`? Seems like that function should be params-agnostic.

Reply:
Yeah I don't think `checkAcceptList` should be mutative. Let's move this into another function or into `SetPrices` directly.